app/test/golden/pagination_first.html

```diff
@@ -0,0 +1,45 @@
+
+<div class="pagination">
+  <ul>
+      <li class="disabled">
+        <a href="/packages?page=0">
+         &laquo
```

Comment:
It should, but for some reason it is working, nevertheless. Fixing it in the subsequent change, where I'm modifying the related template.